sdk/storage/storage-file-datalake/package.json

```diff
@@ -116,54 +92,74 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^2.1.2",
-    "@azure/core-auth": "^1.4.0",
-    "@azure/core-client": "^1.6.2",
+    "@azure/core-auth": "^1.9.0",
+    "@azure/core-client": "^1.9.3",
     "@azure/core-http-compat": "^2.0.0",
-    "@azure/core-paging": "^1.1.1",
-    "@azure/core-rest-pipeline": "^1.10.1",
-    "@azure/core-tracing": "^1.1.2",
-    "@azure/core-util": "^1.1.1",
+    "@azure/core-paging": "^1.6.2",
+    "@azure/core-rest-pipeline": "^1.19.1",
+    "@azure/storage-common": "^1.0.0",
+    "@azure/core-tracing": "^1.2.0",
+    "@azure/core-util": "^1.11.0",
     "@azure/core-xml": "^1.4.3",
-    "@azure/logger": "^1.0.0",
+    "@azure/logger": "^1.1.4",
     "@azure/storage-blob": "^12.26.0",
     "events": "^3.0.0",
-    "tslib": "^2.2.0"
+    "tslib": "^2.8.1"
   },
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
     "@azure-tools/test-perf": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
-    "@azure-tools/test-utils": "^1.0.1",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^4.0.1",
-    "@types/chai": "^4.1.6",
-    "@types/mocha": "^10.0.0",
+    "@azure/identity": "^4.7.0",
     "@types/node": "^18.0.0",
-    "@types/sinon": "^17.0.0",
-    "chai": "^4.2.0",
+    "@vitest/browser": "^3.0.6",
+    "@vitest/coverage-istanbul": "^3.0.6",
     "dotenv": "^16.0.0",
-    "es6-promise": "^4.2.5",
     "eslint": "^9.9.0",
     "execa": "^6.0.0",
```

Comment:
Is this needed?

Reply:
Removed

Reply:
Is the removal coming in a future PR, I still see it.